### PR TITLE
Swagger Documentation of EntityFieldConfigurationApiResources.java and LoanChargesApiResource.java

### PR DESCRIPTION
**Pull request by luckyman20**

### CODE REVIEW

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java [hunk 1]
```diff
@@ -83,7 +75,9 @@ public EntityFieldConfigurationApiResources(final PlatformSecurityContext contex
 	@GET
 	@Consumes({ MediaType.APPLICATION_JSON })
 	@Produces({ MediaType.APPLICATION_JSON })
-	public String getAddresses(@PathParam("entity") final String entityname, @Context final UriInfo uriInfo) {
+    @ApiOperation(httpMethod = "GET", value = "Retrieves the Entity Field Configuration", notes = "It retrieves all the Entity Field Configuration")
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java]: Indentation of 2 is required

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java [hunk 2]
```diff
@@ -83,7 +75,9 @@ public EntityFieldConfigurationApiResources(final PlatformSecurityContext contex
...
+    @ApiResponses({@ApiResponse(code = 200, message = "OK", response = EntityFieldConfigurationApiResourcesSwagger.GetFieldConfigurationEntityResponse.class)})
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java]: Indentation of 2 is required

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java (hunk 1 above)
Comment (luckyman20) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java]: at which lines

**Comment by luckyman20** [comment]
Specify the lines please

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java (hunk 1 above)
Comment (Sanyam96) [reply fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java]: Line 78 and 79

**Comment by Sanyam96** [comment]
Line 78 and 79 only
rest looks good

File: fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java [hunk 3]
```diff
@@ -83,7 +75,9 @@ public EntityFieldConfigurationApiResources(final PlatformSecurityContext contex
 	@GET
 	@Consumes({ MediaType.APPLICATION_JSON })
 	@Produces({ MediaType.APPLICATION_JSON })
-	public String getAddresses(@PathParam("entity") final String entityname, @Context final UriInfo uriInfo) {
+      @ApiOperation(httpMethod = "GET", value = "Retrieves the Entity Field Configuration", notes = "It retrieves all the Entity Field Configuration")
+      @ApiResponses({@ApiResponse(code = 200, message = "OK", response = EntityFieldConfigurationApiResourcesSwagger.GetFieldConfigurationEntityResponse.class)})
```
Comment (Sanyam96) [inline fineract-provider/src/main/java/org/apache/fineract/portfolio/address/api/EntityFieldConfigurationApiResources.java]: indentation

**Comment by Sanyam96** [comment]
Squash into a single commit

**Comment by luckyman20** [comment]
@Sanyam96 Updated

**Pull request merged**